editorconfig.el

```diff
@@ -418,12 +418,15 @@ FILETYPE should be s string like `\"ini\"`, if not nil or empty string."
 (defun editorconfig-call-editorconfig-exec ()
   "Call EditorConfig core and return output."
   (let ((filename (buffer-file-name)))
-    (with-temp-buffer
-      (setq default-directory "/")
-      (if (eq 0
-              (call-process editorconfig-exec-path nil t nil filename))
-          (buffer-string)
-        (error (buffer-string))))))
+    (if (and (file-name-absolute-p filename)
+             (file-exists-p filename))
+        (with-temp-buffer
+          (setq default-directory "/")
```

Comment:
Does this line work on Windows?

Reply:
This line has been merged in https://github.com/editorconfig/editorconfig-emacs/pull/47 , and it seems to be ok.

Reply:
Oh, cool. That's great to know!